fiftyone.devicedetection.onpremise/tests/deviceDetectionOnPremise.test.js

```diff
@@ -196,4 +200,29 @@ describe('deviceDetectionOnPremise', () => {
     expect(count).toBe(20);
     done();
   });
+
+  // Check if dataFileUpdateBaseUrl property are set correctly
+  test('Properties for on-premise engine - dataFileUpdateBaseUrl', done => {
+    server.addListener('request', () => {});
+    server.listen(PORT);
+
+    const customDataFileUpdateUrl = `http://localhost:${PORT}`; // in case of null passed, default update link ll be used
+
+    const pipeline = new FiftyOneDegreesDeviceDetectionOnPremise
+      .DeviceDetectionOnPremisePipelineBuilder({
+        dataFile: DataFile,
+        updateOnStart: true,
+        autoUpdate: true,
+        dataFileUpdateBaseUrl: customDataFileUpdateUrl
+      })
+    // We're just checking for access to our server, the evidence that it happened is non-existing header in response
+    expect(() => pipeline.build()).toThrow(`Invalid character in header content ["If-Modified-Since"]`);
```

Comment:
Let's change this test to rather verify that the server has received a request.  Since this will only fail on the non-English locale